src/python/bcc/__init__.py

```diff
@@ -445,12 +445,12 @@ def get_table(self, name, keytype=None, leaftype=None, reducer=None):
         if map_fd < 0:
             raise KeyError
         if not keytype:
-            key_desc = lib.bpf_table_key_desc(self.module, name)
+            key_desc = lib.bpf_table_key_desc(self.module, name).decode("utf-8")
```

Comment:
Does the JSON library always return UTF-8 or it depends on the system's encoding?